builder/qemu/config.go

```diff
@@ -520,6 +520,45 @@ type Config struct {
 	//  * ARM: tpm-tis-device
 	//  * PPC (p-series): tpm-spapr
 	TPMType string `mapstructure:"tpm_device_type" required:"false"`
+	// This is an array of tuples of boot commands, to type when the virtual
+	// machine is booted. The first element of the tuple is the actual boot
+	// command. The second element of the tuple is a description of what the boot
+	// command does. This is intended to be used for interactive installers that
+	// requires many commands to complete the installation. Both the command and
+	// the description will be printed when logging is enabled. When debug mode is
+	// enabled Packer will pause after typing each boot command. This will make it
+	// easier to follow along the installation process and make sure the Packer
+	// and the installer are in sync. It's recommended to use either `boot_steps`
+	// or `boot_commands`.
```

Comment:
Given that the two are by nature conflicting, I would think we can enforce this behaviour, if one is defined, the other should be empty.

Reply:
Fixed in https://github.com/hashicorp/packer-plugin-qemu/pull/103/commits/4de68fb0ead839d25a695a216eda330f0001fb56.